__tests__/app.test.js

```diff
@@ -43,3 +44,14 @@ describe("GET /api/topics", () => {
     return request(app).get("/api/banana").expect(404);
   });
 });
+
+describe("GET /api", () => {
+  test(" should respond 200 and contents of endpoints.json", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual(data);
+      });
```

Comment:
I'd consider adding another test to check that the JSON object takes the correct form - think about using **toMatchObject** 👍